Projects/Server/World.cs

```diff
@@ -629,7 +629,7 @@ public static void Save(bool message, bool permitBackgroundWrite)
       SaveStrategy strategy = SaveStrategy.Acquire();
       Console.WriteLine("Core: Using {0} save strategy", strategy.Name.ToLowerInvariant());
 
-      Console.Write("World: Saving...");
+      Console.Write($"[{DateTime.Now.ToLongTimeString()}] World: Saving...");

```

Comment:
Should use UtcNow

Reply:
I've got that switched to UTC. Good catch :)